### ISSUE TITLE
Write a new data scraper for getting previous quarter's data

### ISSUE THREAD

**Issue by phi-line** [OpenCourseAPI/OwlAPI#33]
This would involve going to each quarter's previous data and using Beautiful Soup to find gather all the departments, all of the child courses, and all of the child sections. Then, using the same format as the current data scraper, make a database for every quarter.

The resulting database should be named using this schema:

`[year][quarter][campus]_database.json`

> `201812_database.json`
```

    201812: The 2017-2018 academic year runs from July 1, 2017 to June 30, 2018. Therefore the ending year, 2011 is used to refer to the entire year.

    201812: The fifth digit represent the quarter:
        1 for Summer
        2 for Fall
        3 for Winter
        4 for Spring

    201812: The last digit refers to the college, 1 for Foothill and 2 for De Anza. 
```

This would make our local database very consistent with the one found on MyPortal, as the `GET` request made for the tabular course data follows this same format.

**Comment by phi-line** [comment]
As per our discussion @TryExceptElse , I'll be taking this issue. I'll work on trying to reverse engineer the post request for the past quarter's data if possible. There might be a way we can retrieve without the auth cookie.

Additionally, I'll work on getting this to look exactly like our current 'schema' with TinyDB. Another issue will be brought up after this issue resolves laying out how TinyDB can be extended, or the use of another relational database an be used.

**Comment by madhavarshney** [comment]
Solved by #59. Follow up to this is #86.
